flexmeasures/data/services/sensors.py

```diff
@@ -1,43 +1,36 @@
 from __future__ import annotations
 
-from werkzeug.exceptions import NotFound
+import sqlalchemy as sa
 
 from flexmeasures import Sensor, Account
 from flexmeasures.data.models.generic_assets import GenericAsset
 
 
 def get_sensors(
-    account_name: str | None = None,
+    accounts: list[Account | None] = None,
+    filter_by_sensor_ids: list[int] | None = None,
 ) -> list[Sensor]:
-    """Return a list of Sensor objects.
+    """Return a list of Sensor objects that belong to any of the given accounts, and/or public sensors.
 
-    :param account_name: optionally, filter by account name.
+    :param accounts:             optionally, select only sensors from this list of accounts
+                                 (include None to select sensors that belong to a public asset).
```

Comment:
I'm not a fan of the None argument to indicate public assets. 
I believe an explicit parameter "include_public_assets", which defaults to False, would make it easier to see what's going on.